### PR TITLE
Disable `try and catch` for the play button observer

### PR DESCRIPTION
**Pull request by JohnyP36**
> Any additions, changes or removals is at the Authors discretion.  
> You're free to counterargue (to a certain point) if you disagree with the decision.  
> To avoid being banned, don't constantly re-open or create new (related) issue reports or pull requests.  
### Prerequisites

- [x] I've read the [Code of Conduct](https://github.com/JohnyP36/YT-Nonstop/blob/main/.github/CODE_OF_CONDUCT.md) and the [Contributing Guidelines](https://github.com/JohnyP36/YT-Nonstop/blob/main/.github/CONTRIBUTING.md)
- [x] This is more than one or two issue(s)
    - Otherwise open a [new issue](https://github.com/JohnyP36/YT-Nonstop/issues/new/choose)
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- [x] I tried to reproduce the issue when...
    - [x] Other webextensions related to Youtube are disabled 
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of the extension

### Describe the issue
 

See #22 : extension seems to keep reloading by certain users Google/YT blocks using and Adblocker. 

### Browser & Extension version
 
| Browser name & version | Extension Version | Operating System |
| :---                   |       :---:       |             ---: |
| \`**Firefox** 119\`   | \`_**2.0.9.2**_\`| \`**Windows 10**, 22H2\` | 

#### Notes

Basically reverts halve of https://github.com/JohnyP36/YT-Nonstop/commit/4e76d408034759c8f7e1a03f992fa38307ff5b4a. 

Disadvantage: if the extension couldn't find the play button if will simply stop working and you have to manually reload the webpage.

### CODE REVIEW

**Comment by JohnyP36** [comment]
As per issue [#22 Constant Reloading After Ublock/Google Changes to Ads](https://github.com/JohnyP36/YT-Nonstop/issues/22), I can't reproduce this issue. 
Although in the newest version of this extension in the Firefox Add-ons this "problem" got patched, I will not change it in the core code on this repo - as explained earlier.
